app/src/Store.php

```diff
@@ -1,23 +1,51 @@
 <?php
 
 namespace App;
+
 Use App\Book as Book;
+
 /**
  * Class Store - prepare and execute query to DB to get all book names, ids, prices and images
  * @package App
  * @return array(object) of all books
  */
 class Store
 {
+
+    public $db;
+
+    public function __construct(DbConnector $connection)
+    {
+        $this->db = $connection->db;
+    }
+
     /**
      * @param $db - property from DbConnector class
      */
-    public function getData($db){
-        $query = $db->prepare("SELECT `id`, `title`, `price`, `image` FROM `books`;");
+    public function getAllBooks()
+    {
+        $query = $this->db->prepare("SELECT `id`, `title`, `price`, `image` FROM `books`;");
         $query->execute();
         $query->setFetchMode(\PDO::FETCH_CLASS, Book::class);
         $books = $query->fetchAll();
-        var_dump($books);
         return $books;
     }
+
+
+    /**
+     * @param $books - returned from getAllBooks, an array of objects
+     */
+    public function showAllBooks($books)
+    {
```

Comment:
Could not spot unit tests? Did I miss it? When you have them and removed the line-breaks I will also test it manually and merge it.